app/adapters/-intl-adapter.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+import IntlGetResult from 'ember-intl/models/intl-get-result';
+import Locale from 'ember-intl/models/locale';
+import IntlAdapter from 'ember-intl/adapter';
+
+function normalize (fullName) {
+    Ember.assert('Lookup name must be a string', typeof fullName === 'string');
+
+    return fullName.toLowerCase();
+}
+
+export default IntlAdapter.extend({
```

Comment:
Not documenting this yet, but it will eventually be exposed for those people who want to specify their own way of loading translations.  The CLDR portion of this hasn't been thought out yet, but it will ideally be another hook off this.

See: https://github.com/yahoo/ember-intl/issues/69